buildConfiguration.xml

```diff
@@ -2,7 +2,7 @@
   <dotnetArchitecture>x64</dotnetArchitecture>
   <nugetVersion>3.5.0-rc-1285</nugetVersion>
   <runtimes>net45,net461,netstandard2.0</runtimes>
-  <assemblyVersion>6.32.4</assemblyVersion>
+  <assemblyVersion>6.33.0</assemblyVersion>
```

Comment:
6.33.0 has been released, should this be 6.33.1?

Reply:
fixed